kv/kvtls/test/ca.go

```diff
@@ -43,15 +45,17 @@ func newCA(name string) *ca {
 		},
 		serial: big.NewInt(2),
 	}
-
 }
 
 func writeExclusivePEMFile(path, marker string, mode os.FileMode, data []byte) (err error) {
 	f, err := os.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_EXCL, mode)
 	if err != nil {
 		return err
 	}
-	defer runutil.CloseWithErrCapture(&err, f, "write pem file")
```

Comment:
We use this `CloseWithErrCapture()` util quite frequently. We may just copy it into dskit and use it here. It's a tiny util, sounds fine copying it.

Reply:
@pracucci OK looking into it.

Reply:
Done.